### Eclipse Workspace Patch 1.0
#P EGitTestProject
diff --git "a/EGitTestProject/src/Catalogs/\320\241\320\277\321\200\320\260\320\262\320\276\321\207\320\275\320\270\320\272/ObjectModule.bsl" "b/EGitTestProject/src/Catalogs/\320\241\320\277\321\200\320\260\320\262\320\276\321\207\320\275\320\270\320\272/ObjectModule.bsl"
index caa9073..5ea4e46 100644
--- "a/EGitTestProject/src/Catalogs/\320\241\320\277\321\200\320\260\320\262\320\276\321\207\320\275\320\270\320\272/ObjectModule.bsl"
+++ "b/EGitTestProject/src/Catalogs/\320\241\320\277\321\200\320\260\320\262\320\276\321\207\320\275\320\270\320\272/ObjectModule.bsl"
@@ -3,5 +3,5 @@
     c = 12;
     f = -100;
     e = 0;
-    f = 101;
+    f = 102;
 EndProcedure
\ No newline at end of file
